inputFiles/compositionalMultiphaseWell/staircase_co2_wells_3d.xml

```diff
@@ -182,6 +182,13 @@
     <ConstantPermeability
       name="rockPerm"
       permeabilityComponents="{ 2.0e-16, 2.0e-16, 2.0e-16 }"/>
+
+    <ConstantDiffusion
+      name="diffusion"
+      phaseNames="{ gas, water }"
+      defaultPhaseDiffusivityMultipliers="{ 20, 1 }"
+      diffusivityComponents="{ 1e-9, 1e-9, 1e-9 }"/>
```

Comment:
is diffusivity tensor?

Reply:
Yes, I agree with your suggestion. For now, the diffusivity (and the thermal conductivity) use exactly the same terminology as the permeability since they play the same role in the code. That is why I reused the term `components` here. 

But if we all agree that it is a good time to switch from `components` to `tensor`, then let's do it once for all for permeability, thermal conductivity, and diffusivity (in a separate PR possibly). One thing to keep in mind is that this is going to break a lot of xml files.